extension-manifest-v2/app/content-scripts/autoconsent.js

```diff
@@ -12,26 +12,28 @@
 import AutoConsent from '@duckduckgo/autoconsent';
 import { showIframe } from '@ghostery/ui/autoconsent/iframe';
 
-const consent = new AutoConsent(msg => chrome.runtime.sendMessage(
-	{ ...msg, action: 'autoconsent' },
-));
+if (document.contentType === 'text/html') {
+	const consent = new AutoConsent(msg => chrome.runtime.sendMessage(
+		{ ...msg, action: 'autoconsent' },
```

Comment:
how about adding the condition here? diff will be much smaller and the effect the same

Reply:
The diff is effectively two lines (adding one if statement). I don't know how that could be made smaller (or if there is any benefit in moving the if only before the sending?).

Reply:
indentation makes the diff big
<img width="151" alt="Screenshot 2023-03-21 at 14 23 33" src="https://user-images.githubusercontent.com/1228153/226619242-077b921f-d62c-43bc-a29e-c73bbd94d749.png">

Reply:
Github has an option to show diffs without white spaces if that is the concern. It's a small code change.